snuba/request/schema.py

```diff
@@ -133,6 +133,10 @@ def generate_template(self) -> Any:
             "dry_run": {"type": "boolean", "default": False},
             # Flags if this a legacy query that was automatically generated by the SnQL SDK
             "legacy": {"type": "boolean", "default": False},
+            # Flag to retrieve only ClickHouse tracing data. If this is True,
+            # the query execution should not return the results of the query, but
+            # return the tracing data of the query execution within ClickHouse
+            "capture_trace": {"type": "boolean", "default": False},
```

Comment:
I am not sure we want to expose this in the snuba api.
We cannot change the log handler there if that impacts the system logs or snuba logs, which means we would not be using this in that circumstance. 
Also this may not work in HTTP and we want to move away from the TCP protocol for the Snuba queries, which may mean this will not be able to work on the snuba api.

I would suggest we remove it from there till we do not decide to expose it.